documentation/modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -108,65 +124,105 @@ $ sudo podman create \
+. Update the `host_fqdn` environment variable with the fully qualified domain name of the registry node.
++
+[source,bash]
+----
+[user@registry ~]$ host_fqdn=$( hostname --long )
+----
+
+. Update the `b64auth` environment variable with the base64 encoding of the `http` credentials used to create the `htpasswd` file.
++
+[source,bash]
+----
+[user@registry ~]$ b64auth=$( echo -n '<username>:<passwd>' openssl base64 )
+----
++
+Replace `<username>` with the user name and `<passwd>` with the password.
 
+. Set the `AUTHSTRING` environment variable to use the `base64` authorization string. The `$USER` variable is an environment variable containing the name of the current user.
++
+[source,bash]
+----
+[user@registry ~]$ AUTHSTRING="{\"$host_fqdn:5000\": {\"auth\": \"$b64auth\",\"email\": \"$USER@redhat.com\"}}"
 ----
-$ host_fqdn=$( hostname --long )
-$ b64auth=$( echo -n 'dummy:dummy' openssl base64 )
-$ AUTHSTRING="{\"$host_fqdn:5000\": {\"auth\": \"$b64auth\",\"email\": \"$USER@redhat.com\"}}"
 
-$ jq ".auths += $AUTHSTRING" < pull-secret.json > pull-secret.jsoning.new
+. Update the pull-secret file.
++
+[source,bash]
+----
+[user@registry ~]$ jq ".auths += $AUTHSTRING" < pull-secret.json > pull-secret-update.json
 ----
 
-== Mirroring the repository
+
+== Mirroring the repository (optional)
 
 .Procedure
 
-Mirror the remote install images to the local repository.
+. Copy the `oc` binary from the provisioner node to the registry node.
++
+[source,bash]
+----
+[user@registry ~]$ sudo scp kni@provisioner:/usr/local/bin/oc /usr/local/bin
+----
 
+. Mirror the remote install images to the local repository.
++
+[source,bash]
 ----
-$ /usr/local/bin/oc adm release mirror \
-  -a pull-secret.json.new
+[user@registry ~]$ /usr/local/bin/oc adm release mirror \
+  -a pull-secret-update.json
   --from=$UPSTREAM_REPO \
   --to-release-image=$LOCAL_REG/$LOCAL_REPO:${VERSION} \
   --to=$LOCAL_REG/$LOCAL_REPO
 ----
 
-== Modify the `install-config.yaml` file to use the disconnected registry
+== Modify the `install-config.yaml` file to use the disconnected registry (optional)
 
-The `install-config.yaml` file should use the newly created pull-secret.
-The `install-config.yaml` file must also contain the disconnected registry node's certificate and registry information.
+On the provisioner node, the `install-config.yaml` file should use the newly created pull-secret from the `pull-secret-update.json` file. The `install-config.yaml` file must also contain the disconnected registry node's certificate and registry information.
 
 .Procedure
 
 . Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces.
 +
+[source,bash]
 ----
-echo "additionalTrustBundle: |" >> install-config.yaml
-sed -e 's/^/  /' /opt/registry/certs/domain.crt >> install-config.yaml
```

Comment:
This procedure mentions to add the disconnected registry node's ceriticate to the install-config.yaml. However, it makes the assumption that the domain.crt already resides in the provisioner host.

Prior to 
```
[kni@provisioner ~]$ echo "additionalTrustBundle: |" >> install-config.yaml
```
We need a step like the following:
```
[kni@provisioner ~]$ scp user@registry.example.com:/opt/registry/certs/domain.crt /home/kni
```
@iranzo ^ keep me honest, but I believe that is accurate syntax. 

So step 1 should be:
```
[kni@provisioner ~]$ scp user@registry.example.com:/opt/registry/certs/domain.crt /home/kni
[kni@provisioner ~]$ echo "additionalTrustBundle: |" >> install-config.yaml
[kni@provisioner ~]$ sed -e 's/^/  /' /home/kni/domain.crt >> install-config.yaml
```

Reply:
going in next push